tests/__init__.py

```diff
@@ -1,2 +1,10 @@
-LINT = True
+def _have_flake8():
+    try:
+        import flake8
+        return flake8 is not None  # To stop PEP8 from complaining
+    except ImportError:
+        return False
+
+
+LINT = _have_flake8()
```

Comment:
Why making linting optional? For sake of making tests quick, or is it something other?